ui/components/calendar/ReservationCalendarControls.tsx

```diff
@@ -498,9 +498,6 @@ const ReservationCalendarControls = <T extends Record<string, unknown>>({
     <SubmitButtonWrapper>
       <LoginFragment
         isActionDisabled={!isReservable}
-        actionCallback={() => {
```

Comment:
Rollback?

Reply:
yeah, i was afraid of something like this. what should be here?

Reply:
Well, the original content I guess